services/web/client/source/class/qxapp/component/widget/logger/LoggerView.js

```diff
@@ -105,7 +105,7 @@ qx.Class.define("qxapp.component.widget.logger.LoggerView", {
 
       toolbar.add(new qx.ui.toolbar.Separator());
 
-      toolbar.add(new qx.ui.basic.Label("Filter:").set({
+      toolbar.add(new qx.ui.basic.Label(this.tr("Filter") + ":").set({
```

Comment:
```suggestion
      toolbar.add(new qx.ui.basic.Label(this.tr("Filter%1",":")).set({
```
in that way the translation remains 'live' but as said earlier I would use a Placeholder and not a Label for this. And in that case the : is not necessary anyway.